.gitignore

```diff
@@ -35,3 +35,4 @@ original-pom.xml
 **/logback-test.xml
 **/*.swp
 **/dependency-reduced-pom.xml
+compareGeneratedSqlWithExpectedSqlForMinimalChangesets/
```

Comment:
Files do get added into here, but we also have existing files that should be managed as part of git. It's a bit of an odd setup right now until we can clean up those Verify tests, but we can't add this to gitignore